pkg/cli/cmd/credential/register/register.go

```diff
@@ -32,17 +32,19 @@ func NewCommand(factory framework.Factory) *cobra.Command {
 	// This command is not runnable, and thus has no runner.
 	cmd := &cobra.Command{
 		Use:   "register",
-		Short: "Register(Add or update) cloud provider credential for a Radius installation.",
+		Short: "Register (Add or update) cloud provider credential for a Radius installation.",
 		Long:  "Register (Add or update) cloud provider configuration for a Radius installation." + common.LongDescriptionBlurb,
 		Example: `
 # Register (Add or update) cloud provider credential for Azure with service principal authentication
-rad credential register azure --client-id <client id> --client-secret <client secret> --tenant-id <tenant id> 	
+rad credential register azure sp --client-id <client id> --client-secret <client secret> --tenant-id <tenant id>
+# Register (Add or update) cloud provider credential for Azure with workload identity authentication
+rad credential register azure wi --client-id <client id> --tenant-id <tenant id>
 # Register (Add or Update) cloud provider credential for AWS with IAM authentication
 rad credential register aws --access-key-id <access-key-id> --secret-access-key <secret-access-key>	
 `,
 	}
 
-	azure, _ := credential_register_azure.NewCommand(factory)
```

Comment:
Why was the second output removed?

Reply:
since this now calls into `rad credential register azure` (which is a noop command), we don't have a runner. it exists in the subcommands `rad credential register azure sp` and `rad credential register wi` now.